Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ginger"
-version = "0.3.0"
+version = "0.2.3"
```

Comment:
Does Rust/Cargo provide a way to set the version at build time, rather than in code?

Reply:
Not really. You don't have to specify one at all, and if you're baking a string into a binary you can write code that does that. We could do away with version numbers here, and create a build.rs that bakes git ids into the binary.